kuksa_apps/kanto/config/edge_client.ini

```diff
@@ -0,0 +1,11 @@
+[SERVER]
+# host: default 127.0.0.1
+host = 192.168.56.51
+
+[SUBSCRIPTION]
+topic_dev_info = "edge/thing/response"
+topic_dev_info_request = "edge/thing/request"
```

Comment:
These topics are Kanto system and predefined ones and only internally used by any Kanto application that requires the device info Kanto is connected to. They are not configurable, thus, no need to be present in the .ini file. Please, remove them.

Reply:
I removed the section and placed the topics back in `edge_client.py` as in your initial commit.